packages/docs/stories/styles/link-list.stories.ts

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable import/no-relative-packages */
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import {
+  generateScreenshotStory,
+  generateStoryDescription,
+  storybookDefaults,
+} from '../../src/helpers/component.js';
+import { renderStyles } from '../../src/helpers/styles.js';
+import { generateFigmaPluginObject } from '../../src/helpers/figma.js';
+
+const { args, argTypes } = storybookDefaults('syn-link-list');
+
+const generateLinks = (amount = 4, size = 'medium') => new Array(amount)
+  .fill(undefined)
+  .map(() => `
+    <li>
+      <a href="javascript:void()" class="syn-link syn-link--${size}">
+        <syn-icon name="keyboard_arrow_right"></syn-icon>
+        Link
+      </a>
+    </li>
+  `.trim())
+  .join('');
+
+const meta: Meta = {
+  args,
+  argTypes,
```

Comment:
I would remove the default-slot from the docs table. I found it a little bit confusing, as it does nothing. 
see 
![Bildschirmfoto 2024-11-26 um 13 45 02](https://github.com/user-attachments/assets/dd04effd-deaa-4a00-b677-3001dde6e389)


By adding following, we could get rid of the default slot in the table: 
```js
delete args['default-slot'];
delete argTypes['default-slot'];
```
![image](https://github.com/user-attachments/assets/da30654b-b58a-4c04-89af-704366fb8acc)

and then it would look like:
![Bildschirmfoto 2024-11-26 um 13 42 25](https://github.com/user-attachments/assets/c97e7686-4214-4bc3-8329-1c20a71a5a38)